### PR TITLE
[Oneline] - Add type definition 

### PR DESCRIPTION
**Pull request by diogopxrnextday**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:

- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

If removing a declaration:

- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Review by edwinveldhuizen — APPROVED** [review]

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/pxr-oneline/package.json
1. types/pxr-oneline/tsconfig.json
1. types/pxr-oneline/index.d.ts
1. types/pxr-oneline/oneline-tests.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a3cfbe13f5d22c4d4489f6df7f9f6e972146d6d9
</p>

File: types/pxr-oneline/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "node16",
+        "lib": [
+            "es7", "dom"
+        ],
```
Comment (peterblazejewicz) [inline types/pxr-oneline/tsconfig.json:6]:
```suggestion
            "es6",
            "dom"
        ],
```
format. It works with ES6, what's behind ES7 here?

**Comment by peterblazejewicz** [comment]
what's the relation to:
https://www.npmjs.com/package/@pxr-media/oneline

**Comment by diogopxrnextday** [comment]
> what's the relation to: https://www.npmjs.com/package/@pxr-media/oneline

That package is being maintained by us, but our goal is to start using DefinitelyTyped to export the interface.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

**Comment by diogopxrnextday** [comment]
Can this PR be merged then?

**Comment by diogopxrnextday** [comment]
Ready to merge

**Pull request merged**